lib/feedjira/date_time_utilities.rb

```diff
@@ -11,14 +11,16 @@ module DateTimeUtilities
     # Parse the given string starting with the most common parser (default ruby)
     # and going over all other available parsers
     def parse_datetime(string)
-      DATE_PARSERS.find do |parser|
+      res = DATE_PARSERS.find do |parser|
```

Comment:
using abbreviations results from using the rubocop defaults with a line length of 80 chars. In this case it's not nescessary (it will fit exactly) but when adding more code you'll have to refactor.